tasks/interactive-bots.md

```diff
@@ -284,14 +285,14 @@ the `GCI tasks` stream. Consult with mentors and make sure your idea is approved
   * The bot posts the expected output.
 
 * Document your interactive bot at `~/zulip/contrib_bots/lib/<bot_name>` in a
-`docs.md`. Add any relevant screenshots that will help understand how the bot
-works.
+`docs.md` (about the bot, how the bot works, what the bot does, etc.). Add any relevant
+ screenshots that will help understand how the bot works.
 
 * Create a commit with your bot, with commit message `interactive bots: Create
 <bot_name> bot.`.
 
 * Create a pull request in the `zulip/zulip` repository, with title
-`interactive bots: Create <bot_name> bot by <username>.`. Link to your GCI task in comment section.
+`interactive bots: Create <bot_name> bot by <username>.`. Link of your pull request to your GCI task in comment section.
```

Comment:
@abhayKashyap03 @TomaszKolek this is not right - the student was supposed to link their GCI task in their PR.